lib/OpenQA/Utils.pm

```diff
@@ -76,7 +76,7 @@ BEGIN {
     $MARKER_URLS = join('|', keys %BUGURLS);
 
     # <marker>[#<project/repo>]#<id>
-    $BUGREF_REGEX = qr{(?<match>(?<marker>$MARKER_REFS)\#?(?<repo>[a-zA-Z/-]+)?\#(?<id>([A-Z]+-)?\d+))};
+    $BUGREF_REGEX = qr{(?<match>(?<marker>$MARKER_REFS)\#?(?<repo>[a-zA-Z/_-]+)?\#(?<id>([A-Z]+-)?\d+))};
```

Comment:
Maybe it makes more sense to use a negative list of characters here to specify only the characters that should *not* match. Otherwise we'll have the same problem again if someone e.g. puts a `.` in the URL.

Reply:
As there are clear requirements of what should be in that exclusion I suggest to match everything for now, to move forward. What do you think?

Reply:
I guess the `#` needs to be in the exclusion.

Reply:
> Otherwise we'll have the same problem again if someone e.g. puts a `.` in the URL.

That was maybe a very bad example on my side - because on second though we probably want punctuation characters to act as terminators when parsing URLs. So maybe add the `.` to the range (which is now an exclude list).

Reply:
Please take a look into https://stackoverflow.com/a/59082561/22022448 . For security reasons @perlpunk and me would prefer a passlist but I guess we are ok with how it is now